lib/components/Example-wrapper.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react"
```

Comment:
the casing on this file is a bit off, `ExampleWrapper.tsx` or `example-wrapper.tsx`